src/main/java/seedu/staffsnap/logic/parser/ParserUtil.java

```diff
@@ -213,4 +215,31 @@ public static Rating parseRating(String rating) throws ParseException {
     public static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
         return Stream.of(prefixes).allMatch(prefix -> argumentMultimap.getValue(prefix).isPresent());
     }
+
+    /**
+     * Parses a {@code String status} into a {@code Status}.
+     * @param status String representation of Status
```

Comment:
To improve coding standard, can add an empty line between javadoc description and parameter section 👍